<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>424</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>6</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 6pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QWidget" name="wServerStart" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonStopServer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="text">
         <string>Stop Server</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonStartServer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="text">
         <string>Start Server</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="wPath" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16000</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="2" column="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineCollection">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16000</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Your Mod Collection ID or URL</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Your Mod Collection ID or URL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonCollectionCheck">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16000</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Check your Collection and list Mods</string>
           </property>
           <property name="text">
            <string>Check</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkDisableMods">
           <property name="toolTip">
            <string>Disable Mods</string>
           </property>
           <property name="text">
            <string>Disable Mods</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="linePath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16000</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Your Path to your Conan Server / Emtpy Directory for your new Server</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Your Path to your Conan Server</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="buttonPath">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Your global Path to your Servers</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="headerLogo" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>115</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/logo/images/logo.jpg);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>422</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QWidget" name="wServerSettings" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16000</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16000</height>
             </size>
            </property>
            <property name="title">
             <string>Server Start Settings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="horizontalSpacing">
              <number>4</number>
             </property>
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="0" column="3">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Max Players</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Steam Port</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLineEdit" name="lineMaxPlayers">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16000</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Max Player Slots</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Server Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineServerName">
               <property name="placeholderText">
                <string>Server Name</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Server Port</string>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="QLineEdit" name="lineServerPort">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Server Port</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QLineEdit" name="lineSteamPort">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Steam Port</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineServerRestart">
               <property name="toolTip">
                <string>Restart every X Minutes</string>
               </property>
               <property name="placeholderText">
                <string>Restart every X Minutes</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Server Restart</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Admin Password</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineAdminPW">
               <property name="placeholderText">
                <string>Admin Password</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonSave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16000</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Save your Serverconfig</string>
        </property>
        <property name="text">
         <string>Save Config</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonInstallServer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16000</width>
          <height>20</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Installs or Updates a Server</string>
        </property>
        <property name="text">
         <string>Install  / Update Server</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
